diff --git a/node_modules/@react-three/fiber/dist/index.js b/node_modules/@react-three/fiber/dist/index.js
index 92b15f9..a3c5a77 100644
--- a/node_modules/@react-three/fiber/dist/index.js
+++ b/node_modules/@react-three/fiber/dist/index.js
@@ -10,7 +10,7 @@ var import_react15 = require("react"), import_react_dom = require("react-dom");
 var import_react_reconciler = __toESM(require("react-reconciler"));
 
 // ../../node_modules/its-fine/dist/index.mjs
-var import_react = require("react");
+var import_react = __toESM(require("react"), 1);
 var e = Error, n = (t) => "function" == typeof t;
 function r(t) {
   let r2 = null;
@@ -42,7 +42,7 @@ var u = r(void 0);
 var d = (t) => t.type;
 
 // ../../node_modules/@react-three/fiber/dist/chunk-KJL4NA4W.mjs
-var import_react2 = require("react");
+var import_react2 = __toESM(require("react"), 1);
 var is = {
   obj: (a) => a === Object(a) && !is.arr(a) && typeof a != "function",
   fun: (a) => typeof a == "function",
@@ -64,7 +64,7 @@ function useMutable(data) {
 var _a;
 
 // ../../node_modules/@react-three/fiber/dist/chunk-F544T326.mjs
-var import_react4 = require("react");
+var import_react4 = __toESM(require("react"), 1);
 var import_scheduler = require("scheduler");
 var import_three = require("three");
 var import_its_fine = require("its-fine");
@@ -957,7 +957,7 @@ var createStore = (callback, MIP, CI, OP) => {
 };
 
 // ../../node_modules/@react-three/fiber/dist/chunk-23M5A6OK.mjs
-var import_react5 = require("react");
+var import_react5 = __toESM(require("react"), 1);
 var import_react_reconciler = require("react-reconciler");
 var import_three2 = require("three");
 var import_its_fine2 = require("its-fine");
@@ -2160,7 +2160,7 @@ var events = {
 };
 
 // ../../node_modules/@react-three/fiber/dist/chunk-OI44446J.mjs
-var import_react14 = require("react");
+var import_react14 = __toESM(require("react"), 1);
 var import_three5 = require("three");
 var import_its_fine3 = require("its-fine");
 var import_react_reconciler2 = require("react-reconciler");
diff --git a/node_modules/@react-three/fiber/dist/index.mjs b/node_modules/@react-three/fiber/dist/index.mjs
index 9239854..f7943d0 100644
--- a/node_modules/@react-three/fiber/dist/index.mjs
+++ b/node_modules/@react-three/fiber/dist/index.mjs
@@ -1,5 +1,5 @@
 // ../../node_modules/zustand/esm/vanilla.mjs
-import { createStore } from "zustand/vanilla";
-export * from "zustand/vanilla";
+import {createStore} from "zustand/vanilla";
+export * from "zustand/vanilla";
 
 // ../../node_modules/@react-three/fiber/dist/index.mjs
 export * from "@react-three/fiber";
-//# sourceMappingURL=index.mjs.map
\ No newline at end of file
+//# sourceMappingURL=index.mjs.map
